sceptre/scipool/config/bmgfki/sc-product-assoc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: "sc-ec2-action-associations.yaml"
+stack_name: "sc-product-assoc-ec2-linux-jumpcloud-notebook"
```

Comment:
The name suggests that this product is accessed through JumpCloud.  Is that what we want for an S.C. instance set up for external users?

Reply:
no, it's just the name name of the cloudformation stack.  we named it that originally because we used jumpcloud to access instances.  I've just copied it over to this new setup.  The SC end user will not see any references to `jumpcloud`.